docs/rules/sort-comp.md

```diff
@@ -70,30 +70,36 @@ The default configuration is:
       'defaultProps',
       'constructor',
       'getDefaultProps',
-      'getInitialState',
```

Comment:
this could technically be a breaking change; is there a reason to swap the order of state and getInitialState?

Reply:
I changed the order in the documentation to match the actual order in the code, I didn't actually change the order in the code. Also the two are mutually exclusive in actual usage so testing them together would be strange, and I wouldn't be sure how to even write that test considering that `state` only exists as an ES6 `Class` property and `getInitialState` only exists as a `createReactClass` method.

Reply:
Makes perfect sense :-) I didn't notice this was just the docs.